### DISCUSSION TITLE
Blazor-server app uses a fallback authorization strategy, In the Login.razor file, I used the @Attribute [AllowAnonymous] attribute, but I still can't interact with it. I can't enter my email or password, and the login button doesn't respond.Is there something wrong with it?

### DISCUSSION THREAD

**Discussion by zuijiaoghb** [microsoft/fluentui-blazor#2833] [Q&A]
[jtpjsapp.zip](https://github.com/user-attachments/files/17430912/jtpjsapp.zip)

# 🐛 Bug Report

Using the revoke authorization strategy, the Login.razor file is not rendering properly.

## 💻 Repro or Code Sample
Program.cs
![Image](https://github.com/user-attachments/assets/5b577d1a-335a-4b9d-ace2-65069ac23400)

Login.razor
![Image](https://github.com/user-attachments/assets/b41dcba3-df31-40bb-af17-9703c1062842)

## 🤔 Expected Behavior

![Image](https://github.com/user-attachments/assets/b56efdfc-9eaf-4a21-8512-4496cb3e6553)

## 🌍 Your Environment

centos8 , PC
* Browser Microsoft Edge
* .NET 8.0.301 and Fluent UI Blazor library Version 4.10.2

**Comment by vnbaaij** [comment]
>Using the revoke authorization strategy, the Login.razor file is not rendering properly.

You **really need** to be more prescriptive on what the issue is. Just that one sentence does not make it clear to us what the problem is. You did not change anything from your other issue report other than putting in a zip file of the reproduction code (without ANY explanation). Also, we still don't understand the language in the screenshot. This does **not** tell us what the expected behavior is.

Do you expect us to just figure out by ourselves what the issue is and hand you a solution ? Sorry, but open source **does not** work that way.  You need to help us to help you.

I will not look at this until the details,, problem, etc are properly described. If not provided, this one will be closed as well.

**Comment by zuijiaoghb** [comment]
Blazor-server app using Microsoft.AspNetCore.Identity;using Microsoft.AspNetCore.Authorization; uses a fallback authorization strategy, In the Login.razor file, I used the @Attribute [AllowAnonymous] attribute, but I still can't interact with it. I can't enter my email or password, and the login button doesn't respond.Is there something wrong with it?

**Comment by vnbaaij** [comment]
The authorization pages need to be in SSR mode. Have you checked that?

**Comment by zuijiaoghb** [comment]
![Image](https://github.com/user-attachments/assets/0a55fbf6-4bee-4b5f-bf4c-dd12d0fe5479)

The blazor server app is in SSR mode, and interactive server-side rendering (interactive SSR) is enabled by the "server" option. Is that the configuration?

**Comment by zuijiaoghb** [comment]
![Image](https://github.com/user-attachments/assets/850093ee-ae21-475c-88e5-47494024393a)

The Login.razor page is also configured with @rendermode InteractiveServer。

**Comment by vnbaaij** [comment]
Look at how the pages are configured when creating a site with authentication through the template. The authentication pages need to be SSR (which in Blazor means NOT having @rendermode! ) 

Moving this to the discussion area as it is not a problem with the library but with your code/configuration. I think you need to study up a bit more on the .NET 8 rendermodes
